Adding custom font and colors assists 🎨

ziadali22 opened the pull request.

ziadali22 committed:
adding custom font and colors assests

ahmdmhasn commented on Dokan/Dokan/Assets.xcassets/Blue Ocean.colorset/Contents.json:
```diff
@@ -0,0 +1,38 @@
+{
```
Do we really need all of these colors?

ahmdmhasn commented on Dokan/Dokan/Info.plist:
```diff
@@ -2,6 +2,12 @@
 <dict>
+	<key>UIAppFonts</key>
+	<array>
+		<string>DMSans-Regular.ttf</string>
```
Did you heard about dynamic font? 😁

ahmdmhasn reviewed (commented):
🚀

ahmdmhasn reviewed (approved):
🚀🚀🚀

ahmdmhasn commented:
We may merge this directly into `develop` since it only contains colors and fonts resources. 
@ziadali22

ziadali22 committed:
Squashed commit of the following:

commit 575cf24da0f3fb5a82f95d5e081e6bcb6ef0b7b0
Author: RaniaZeid <67391043+RaniaZeid@users.noreply.github.com>
Date:   Fri Jul 22 17:43:31 2022 +0200

    feat: Update Swift version for `Domain` and `Core` (#58)

    update swift version at domain and core

commit 3f4aa7c10ffeffc929bce7d602ff53295978e03a
Author: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>
Date:   Wed Jul 20 12:15:43 2022 +0200

    fix: project build error due to a conflict issue ⚙️ (#63)

    fix: project build error due to a conflict issue

commit 3dd7d39d64548472370680cd927a1e903423aca9
Author: claramounir <107275541+claramounir@users.noreply.github.com>
Date:   Tue Jul 19 04:22:35 2022 +0200

    feat/ Create Login View Controller 🎃 (#48)

    * Setup Login (template & unit test)

    * Add basic design to LoginViewController

    Add basics UI components to the xib file and adjust Autolayout to each component to fit in any device (iphones - ipads)

    * Update LoginViewController.swift

    * add-view-model-input-output-functions

    * Bind ViewModel to view

    * Update DokanTests.swift

    * Adjust LoginViewModelTests

    * Update LoginViewController.xib

    * adjust IBOutlet names

    * Create LoginViewControllerTests.swift

    * Update Dokan/Dokan/Scenes/Login/LoginViewController.swift

    Co-authored-by: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>

    * Update Dokan/Dokan/Scenes/Login/LoginViewModelType.swift

    Co-authored-by: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>

    * Update LoginViewController.swift

    * Update LoginViewController.swift

    * Update LoginViewModel.swift

    * Update LoginViewModelTests.swift

    * Enhance UI & Constrains

    * Update LoginViewController.swift

    * Update LoginViewController.swift

    * Adjust some constrains

    Co-authored-by: omarradwan <oradwan037@gmail.com>
    Co-authored-by: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>
    Co-authored-by: Ahmad Yasser <42544598+ahmdyasser@users.noreply.github.com>

commit bbae704d6102670e0455f63110b81e07805f0b4b
Author: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>
Date:   Tue Jul 19 02:35:15 2022 +0200

    Update README.md

commit 55893e9f9ab0771300b9e6d939ac39356079814f
Author: Heba MAD <71231086+HebaMad@users.noreply.github.com>
Date:   Mon Jul 18 13:17:42 2022 +0300

    feat: Add Registration View Controller ⚡️ (#43)

commit de2027557a7cee97a2a7b5bf702e09d094f4666e
Author: Ahmed Nagy <ahmednaagy97@gmail.com>
Date:   Sun Jul 17 11:03:33 2022 +0200

    Make the OTCTextField accessible from outside UIDokan package (#55)

    * Add Verification-Scene to Dokan Target

    * replace the default view controller with the mvvm template files

    * create reusable views folder with the OTCTextfield class

    * make OTCTextField class accessible from outside the UIDokan package

commit a581bb94d133580e5e22ecb9548d615d41d2966d
Author: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>
Date:   Sat Jul 16 12:13:58 2022 +0200

    feat: Add product details view controller and view model 🤺 (#56)

    * Add product details view controller and view model

    * Add scroll view and stack view

    * Add views group

    * Add dummy views

commit 68daed8ffac09d3ffd6fd62ce2f6c1cc34abf781
Author: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>
Date:   Thu Jul 14 03:51:11 2022 +0200

    feat: Add `ShipBookSDK` Logger 📚 (#49)

    * Limit project packages to iOS 13

    * Add `ShipBookSDK`

    * Rename `DefaultLogger` to `ShipBookLogger`

    * Dokan internal loggers will use default logger by default

    * Create group to hold all loggers

commit 9528412f34ed5ee7f7424ed18a388265f7ef923a
Author: Ahmed Nagy <ahmednaagy97@gmail.com>
Date:   Tue Jul 12 06:44:10 2022 +0200

    Add Verification-Scene with custom OTCTextField to Dokan Target (#40)

    * Add Verification-Scene to Dokan Target

    * replace the default view controller with the mvvm template files

    * create reusable views folder with the OTCTextfield class

commit bdab60de08ed5ee6ad3bb7e6d75480c17ccf3d13
Author: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>
Date:   Fri Jul 8 13:54:17 2022 +0200

    Enable Auto Pull Request Reviewers Assign 🏃‍♂️ (#50)

    add code owners to allow auto pull request assign

commit 188db6501d26348e1869e19b461fa2c5dd72b900
Author: Heba MAD <71231086+HebaMad@users.noreply.github.com>
Date:   Wed Jul 6 14:50:25 2022 +0300

    feat: Building domain layer (#35)

    * AddDomainLayer

    * Domain layer

    * DomainLayer

    * NetworkingLayer

    * Revert "NetworkingLayer"

    This reverts commit 6962618164e62bab714433699f1ad6c5fa497424.

    * add domain layer

    * remove duplocated domain

    * add domain layer

    * Update .gitignore

    Co-authored-by: Ahmed M. Hassan <45182214+ahmdmhasn@users.noreply.github.com>

ziadali22 committed:
Merge branch 'develop' into feat/add_fonts_colors

ziadali22 committed:
remove all colors

because in swiftGen branch i create Colors xcassets that will contain all Colors
